app/models/server_health_check_rails/health_check.rb

```diff
@@ -1,7 +1,7 @@
 module ServerHealthCheckRails
   class HealthCheck
     def initialize(*checks)
-      @check = ServerHealthCheck.new
+      @check = ServerHealthCheck.new(logger: Logger.logger)
```

Comment:
My goal in this PR is to handle all health-check related configuration behind the scenes.

Since using a different logger is part of that, passing that logger on to ServerHealthCheck is very much a role that this gem should handle.  As I continue reviewing your comments, I assume I will see that you addressed this point in logger.rb, so let's discuss it there instead of here.